roles/check_resource/README.md

```diff
@@ -15,3 +15,33 @@ resource\_to\_check         | Yes       | MachineConfigPool      | Name of the r
 check\_wait\_retries        | Yes       | Undefined              | Number of times in which the wait task is performed.
 check\_wait\_delay          | Yes       | Undefined              | Time spent between wait tasks' iterations.
 check\_reason               | No        | Undefined              | Reason for the check to be done.
+
+## Requirements
+
+A running OpenShift cluster with the proper credentials is required, credentials must be passed as by setting the KUBECONFIG environment.
+
+## Example of usage
```

Comment:
Hi Beto, I'm a bit confused here. Doesn't it mean we need to pass the following when calling check_resource?

```
    apply:
      environment:
        KUBECONFIG: "{{ kubeconfig_path }}"
```

Because I can see this in the storage_tester example, but not here.

If this is true, this would require to update all the calls to check_resource to pass the KUBECONFIG (in the agent, in partners' hooks like SPK, etc.)

Reply:
Hello Ramon, sorry about the confusion. I removed that var from the storage_tester role. 

We do not need to update the check_resources calls as we are indirectly passing the KUBECONFIG level at each DOA stage: https://github.com/redhat-cip/dci-openshift-agent/blob/84a991f1b927a8358655c1d63066ab37bc708d69/dci-openshift-agent.yml#L419-L423 

That is inherited in roles, plays hooks, etc. But let me do testing with VCP jobs. 

The requirement advice in the READMEs to define that variable should be enough if someone wants to use the role without DOA.